hooks/pre/30-upgrade.rb

```diff
@@ -164,6 +171,7 @@ def fail_and_exit(message)
     upgrade_step :mark_qpid_cert_for_update
     upgrade_step :migrate_candlepin, :run_always => true
     upgrade_step :remove_gutterball
+    upgrade_step :delete_event_queue
     upgrade_step :start_tomcat, :run_always => true
```

Comment:
Why do we have to delete the queue in the hook here? So Puppet creates it again with only specific bindings we want? Or some other reason?